scripts/biomedical/NCBI_Taxonomy/test_data/division_enum_expected.mcf

```diff
@@ -0,0 +1,78 @@
+Node:dcid:BiologicalTaxonomicDivisionEnum
+name: "BiologicalTaxonomicDivisionEnum"
```

Comment:
make sure the name here matches the name of the output file generated by your script. Currently this file is named "ncbi_taxonomy_schema_enum.mcf" if you run the scripts in this import.